src/appleseed/renderer/kernel/lighting/lightsampler.h

```diff
@@ -32,6 +32,8 @@
 
 // appleseed.renderer headers.
 #include "renderer/kernel/intersection/intersectionsettings.h"
+#include "renderer/kernel/lighting/lighttree.h"
+#include "renderer/kernel/lighting/lighttypes.h"
```

Comment:
It will compile without it only because the `lighttree.h` includes it. But we need it for
creating member vectors of `NonPhysicalLightInfo` and `EmittingTriangle`